src/main/java/domain/Player.java

```diff
@@ -0,0 +1,30 @@
+package domain;
+
+public class Player {
+
+    private static final int MAX_LENGTH = 5;
```

Comment:
MAX_LENGTH가 name의 대한 최대 길이인지 명시하는 것은 어떻게 생각하시나요?
```suggestion
    private static final int MAX_NAME_LENGTH = 5;
```

Reply:
좋은것같습니다!